test/SeedCppCoreUnitTest/Tests/DAL/Translators/Db/AddressDbTranslatorTest.cpp

```diff
@@ -0,0 +1,88 @@
+#include "stdafx.h"
+#include "DatabaseTranslatorTest.h"
+
+#include "SeedCppCore/DAL/Translators/Db/AddressDbTranslator.h"
+#include "SeedCppCore/Model/Address.h"
+
+
+using namespace systelab::db::test_utility;
+
+namespace seed_cpp { namespace unit_test {
+
+	struct Address1Wrapper
+	{
+		typedef model::Address ModelEntity;
+
+		static model::Address getEntity()
+		{
+			model::Address address;
+			address.setId(123);
+			address.setCoordinates("N 123.123, E 245,245");
+			address.setStreet("C/Major, 17 2n 3a");
+			address.setCity("Granollers");
+			address.setZip("17246");
+
+			return address;
+		}
+
+		static std::vector<StubFieldValue> getRecordData()
+		{
+			return	{ StubFieldValue("id", 123),
+					  StubFieldValue("patientId", std::string("1A4390F4-C0F6-4827-BC2E-1619BBF56AF6")),
```

Comment:
I'm more fan of using _std::string_literals_ to do something like:
```
StubFieldValue("city", "Granollers"s)
```
Another option would be to have StubFieldValue constructors set as **explicit** so no implicit conversion is allowed.

Reply:
Updated to use std::string_literals as suggested.
The *explicit* option is fancy, but that would imply modifying the StubFieldValue class which belong to another library/repository. As this is out of the scope of this PR, kept as it is for now and considered when further changes of this library are required.